### PR TITLE
common code cleanup part 3

### PR DESCRIPTION
**Pull request by commy2**
ref: https://github.com/acemod/ACE3/pull/2012/files
- [ ] test checkPBO on dedi. RPT dedi

### CODE REVIEW

**Comment by commy2** [comment]
- [ ] test checkPBO on dedi. RPT dedi

**Comment by MikeMatrix** [comment]
You should start referencing these properly.

#2012

File: addons/common/functions/fnc_createOrthonormalReference.sqf [hunk 1]
```diff
@@ -1,19 +1,21 @@
 /*
  * Author: esteldunedain
- *
  * Returns a orthonormal system of reference aligned with the supplied vector
  *
  * Argument:
```
Comment (jokoho48) [inline addons/common/functions/fnc_createOrthonormalReference.sqf]: wrong Arguments
```
 * Arguments: 
 * Vector to align the coordinate system with <ARRAY>
```

File: addons/common/functions/fnc_codeToString.sqf [hunk 2]
```diff
@@ -1,22 +1,23 @@
 /*
  * Author: commy2
- *
  * Removes the brackets around a code and returns the code as a string. It does nothing if the code is already a string.
  *
  * Argument:
```
Comment (jokoho48) [inline addons/common/functions/fnc_codeToString.sqf]: `Arguments:`

File: addons/common/functions/fnc_debug.sqf [hunk 3]
```diff
@@ -1,51 +1,43 @@
...
-
-    _prefix = switch (_level) do {
-        case 0: { "ACE Error" };
-        case 1: { "ACE Warn" };
-        case 2: { "ACE Debug" };
-        case 3: { "ACE Info" };
-        default { "ACE Unknown" };
+    private ["_prefix", "_message"];
+
+    switch (_level) do {
```
Comment (jokoho48) [inline addons/common/functions/fnc_debug.sqf]: Check Posible use 

```
_prefix = ["Error", "Warn", "Debug", "Info"] select _level;
if (isNil "_prefix") then {
    _prefix = "Unkown";
};
```

Comment (commy2) [inline addons/common/functions/fnc_debug.sqf]: No. Thats what default already does.

Comment (jokoho48) [inline addons/common/functions/fnc_debug.sqf]: yeah but select is faster than switch

File: addons/common/functions/fnc_codeToString.sqf (hunk 2 above)
Comment (commy2) [inline addons/common/functions/fnc_codeToString.sqf]: k done here:
https://github.com/acemod/ACE3/commit/a8ba96a01b6db14548981fd8627127d75aaa8eac

File: addons/common/functions/fnc_createOrthonormalReference.sqf (hunk 1 above)
Comment (commy2) [inline addons/common/functions/fnc_createOrthonormalReference.sqf]: k done here:
https://github.com/acemod/ACE3/commit/a8ba96a01b6db14548981fd8627127d75aaa8eac

File: addons/common/functions/fnc_debug.sqf (hunk 3 above)
Comment (commy2) [inline addons/common/functions/fnc_debug.sqf]: Yours throws an error if `_level` is large enough. :stuck_out_tongue:

Comment (SilentSpike) [inline addons/common/functions/fnc_debug.sqf]: `_prefix = ["Error", "Warn", "Debug", "Info"] select (_level min 3);`

Comment (commy2) [inline addons/common/functions/fnc_debug.sqf]: k will do in part 5

**Pull request merged**